packages/autocomplete-html/update/update.js

```
@@ -0,0 +1,289 @@
+  are objects that can contain several valid keys.
+  - global: Seems to be used exclusively for Global Attributes. Is a boolean
+            which when false, the key does not appear.
+  - type: A ?type? for the attribute. It's meaning is not immediately known.
+          Nor a way to reliabley programatically collect it. Some discovered values:
+          * cssStyle: Exclusively used for `class` attribute
+          * boolean: Attributes that only accept `true` or `false`
+          * flag: For attributes that don't require or accept values. eg autoplay
+          * cssId: Exclusively used for the `id` attribute
+          * color: Exclusively used for the `bgcolor` attribute
+          * style: Exclusively used for the `style` attribute
+  - description: A text description of the attribute
+  - attribOption: A string array of valid values that can exist within the attribute.
+                  Such as the case with `rel` where only so many valid options exist.
+  ==============================================================================
+  ==============================================================================
+  A quick check in for the structure of `@webref/elements` `listAll()` return.
+  This will return an object, where each top level key is a HTML spec name.
+  Then within each will be some information on the spec, and `elements` an array
+  of objects that will contain the `name` of our element, and it's interface name.
+  {
+    "css-masking-1": {
+      "spec": {},
+      "elements": [
+        { "name": "clipPath", "interface": "SVGClipPathElement" },
+        { "name": "mask", "interface": "SVGMaskElement" }
+      ]
+    }
+  };
+
+  The `interface` here should match a return that we can receive from DevTools data.
+  ==============================================================================
+  ==============================================================================
```

I honestly would prefer these to become a test instead of a comment...

---

Yeah I did write a whole lot here.

But I can keep some comments to explain what can't be done by a test maybe? I'll look at adding Joi to this so it fails early if we get an unexpected format. But thanks for the suggestion!